AppCenterCrashes/AppCenterCrashesTests/MSErrorLogFormatterTests.mm

```diff
@@ -248,7 +248,7 @@ - (void)testAnonymizedPathWorks {
              @"Containers/Bundle/Application/8CC7B5B5-7841-45C4-BAC2-6AA1B944A5E1/Puppet.app/Puppet";
   actual = [MSErrorLogFormatter anonymizedPathFromPath:testPath];
   assertThat(actual, equalTo(expected));
-  XCTAssertFalse([actual containsString:@"someone"]);
+  XCTAssertFalse([actual hasPrefix:@"/Users/someone"]);
```

Comment:
way better!

Reply:
Yeah.. `hasPrefix:` is available macOS 10.10 and above. This will eliminate compile warning.